kubernetes/charts/weblogic-operator/values.yaml

```diff
@@ -159,6 +159,12 @@ javaLoggingFileCount: 10
 # the operator's deployment. See https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
 # for more information on node selectors.
 #nodeSelector:
+ 
+# webhookOnly - Only install the webhook and skip deploying the operator.
```

Comment:
Change the description of these two fields to be similar to how the descriptions of other fields are worded, e.g. "webhoookOnly specifies that ..."

Reply:
Changed.